src/libraries/System.IO.Ports/tests/SerialPort/ReadTo.cs

```diff
@@ -25,6 +25,9 @@ public class ReadTo : PortsTest
         private const int MIN_NUM_NEWLINE_CHARS = 1;
         private const int MAX_NUM_NEWLINE_CHARS = 5;
 
+        // The web name of Encoding.UTF7
+        private const string UTF7_WEBNAME = "utf-7";
```

Comment:
Can we use our standard casing for consts?

Reply:
I'm following the convention already present in this file.

Reply:
I am seeing this constant value defined in other file too. can we define it in one place and reference it from there instead of defining it twice? 

---
In reply to: [436877102](https://github.com/dotnet/runtime/pull/37535#discussion_r436877102) [](ancestors = 436877102)

Reply:
This field no longer exists as of newest iteration.